internal/postgres/config.go

```diff
@@ -100,6 +103,12 @@ func Environment(cluster *v1beta1.PostgresCluster) []corev1.EnvVar {
 			Name:  "PGPORT",
 			Value: fmt.Sprint(*cluster.Spec.Port),
 		},
+		// Setting the KRB5_CONFIG for kerberos
+		// - https://web.mit.edu/kerberos/krb5-devel/doc/admin/conf_files/krb5_conf.html
+		{
+			Name:  "KRB5_CONFIG",
+			Value: fmt.Sprintf("%s/krb5.conf", configMountPath),
```

Comment:
⛏️ Go recognizes "constant + constant" as a constant, so it's generally better to use over `fmt` in these cases.

```suggestion
			Value: configMountPath + "/krb5.conf",
```